adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -1236,24 +1237,29 @@ func genRouteCreateParams(swagger *model.MgwSwagger, resource *model.Resource, v
 	prodClusterName string, sandClusterName string, requestInterceptor map[string]model.InterceptEndpoint,
 	responseInterceptor map[string]model.InterceptEndpoint, organizationID string) *routeCreateParams {
 	params := &routeCreateParams{
-		organizationID:      organizationID,
-		title:               swagger.GetTitle(),
-		apiType:             swagger.GetAPIType(),
-		version:             swagger.GetVersion(),
-		vHost:               vHost,
-		xWSO2BasePath:       swagger.GetXWso2Basepath(),
-		AuthHeader:          swagger.GetXWSO2AuthHeader(),
-		prodClusterName:     prodClusterName,
-		sandClusterName:     sandClusterName,
-		endpointBasePath:    endpointBasePath,
-		corsPolicy:          swagger.GetCorsConfig(),
-		resourcePathParam:   "",
-		resourceMethods:     getDefaultResourceMethods(swagger.GetAPIType()),
-		requestInterceptor:  requestInterceptor,
-		responseInterceptor: responseInterceptor,
-		rewritePath:         "",
-		rewriteMethod:       false,
-	}
+		organizationID:              organizationID,
+		title:                       swagger.GetTitle(),
+		apiType:                     swagger.GetAPIType(),
+		version:                     swagger.GetVersion(),
+		vHost:                       vHost,
+		xWSO2BasePath:               swagger.GetXWso2Basepath(),
+		AuthHeader:                  swagger.GetXWSO2AuthHeader(),
+		prodClusterName:             prodClusterName,
+		sandClusterName:             sandClusterName,
+		endpointBasePath:            endpointBasePath,
+		corsPolicy:                  swagger.GetCorsConfig(),
+		resourcePathParam:           "",
+		resourceMethods:             getDefaultResourceMethods(swagger.GetAPIType()),
+		requestInterceptor:          requestInterceptor,
+		responseInterceptor:         responseInterceptor,
+		rewritePath:                 "",
+		rewriteMethod:               false,
+		// negation is performing to match the envoy config name (disable_request_body_buffering)
+		isRequestBodyBufferDisabled: !swagger.GetXWso2RequestBodyPass(),
```

Comment:
Are there any drawbacks if we use the positive form? something like `isRequestBodyBuffered`?

Reply:
Change the field name  `isRequestBodyBufferDisabled`